.github/workflows/snyk.yml

```diff
@@ -0,0 +1,24 @@
+
+name: Snyk
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: snowplow-android-tracker
+      
+    - name: Run Snyk to check for vulnerabilities in tracker
+      uses: snyk/actions/gradle@master
+      with:
+        command: monitor
+        args: --all-sub-projects --configuration-matching='debugAndroidTestRuntimeClasspath'
```

Comment:
I'm a bit confused by the configuration. In this case it checks also dependencies used just for testing (e.g. mockWebServer).
Wouldn't `releaseRuntimeClasspath` be more suitable as configuration?

Reply:
Yeah, my initial runs were with `releaseRuntimeClasspath`. 
I've bounced between if we should include development dependencies or not. We don't for the JavaScript libraries so maybe we shouldn't here either.